mks/operations.py

```diff
@@ -57,6 +58,16 @@ def get_brp():
         return log_and_generate_response(e)
 
 
+def get_brp_raw():
+    cookie_value = request.cookies.get('access_token', None)
```

Comment:
`.get('access_token', None)` kan je verkorten naar `.get('access_token')`. Default is `None`.